src/arrayMethodReduce.js

```diff
@@ -5,8 +5,22 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulated = initialValue;
```

Comment:
as you said, variable name must start with noun, not a verb or adjective, right?

Reply:
I think you are rigth here, will rename it 👍